plonetheme/barceloneta/theme/less/header.plone.less

```diff
@@ -59,19 +59,20 @@
         max-width: 50%;
         padding-top: 0;
         position: relative;
-        width: 250px;
+        width: 260px;
         .searchSection {display: inherit;} //yes for non mobile
         [type="text"] {
             border-bottom-right-radius: 0;
             border-top-right-radius: 0;
             float: left;
-            width: 180px; //searchboxwidth (250px) - submit (70px)
+            width: 170px; //searchboxwidth (250px) - submit (70px)
         }
         [type="submit"] {
             display: inline-block;
             margin-left: -1px;
             vertical-align: top;
-            width: 70px;
+            width: 90px;
```

Comment:
On adding width auto, this happens:
![auto](https://cloud.githubusercontent.com/assets/17498833/14304049/38c56c44-fbcd-11e5-9e01-32a17f1f216f.png)
button goes beyond the horizontal hr below, for longer texts and stays behind it for smaller texts.